spec/ingest.yaml

```diff
@@ -0,0 +1,460 @@
+openapi: 3.0.0
+info:
+  title: ingest
+  version: '1.0'
+  contact:
+    email: fantian@360.cn
+paths:
+  '/dae/v1/ingest/{table}':
+    parameters:
+      - schema:
+          type: string
+        name: table
+        in: path
+        required: true
+        description: table to insert data into
+    post:
```

Comment:
I didn't see the common HTTP headers mentioned in this swagger API spec. Can you check the DDL API spec ?